src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -23,7 +23,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 {% set rowIsDisplayed =  (productIndex is defined and paginationNum is defined and productIndex > paginationNum) %}
-<tr id="orderProduct_{{ product.orderDetailId }}"{% if rowIsDisplayed %} class="d-none"{% endif %}>
+<tr id="orderProduct_{{ product.orderDetailId }}" data-product-id="{{ product.orderDetailId }}" data-product-quantity="{{ product.quantity }}" {% if rowIsDisplayed %} class="d-none"{% endif %}>
```

Comment:
I chose to integrate all the data required in javascript in the quantity input field, since it's the only that is common to all features it allows to have a consistent management

However, although I think it's clearer to have all the data on the same DOM element, storing this data in a more "parent" element makes sense (if one day we get rid or toggle the quantity input it could be bothersome), but then maybe in an element more related to the feature Either the cancel product `td`, or the `.cancel-product-cell-elements` div (see https://github.com/PrestaShop/PrestaShop/pull/17416/files#diff-6a8b04b47a9ab83236551aa8ec626e19R108)

Reply:
ok, to be done once your standard refund PR is merged.

Reply:
done